genai-perf/README.md

```diff
@@ -702,13 +713,13 @@ The mean number of turns per session.
 The standard deviation of the number of turns per session.
 (default: `0`)
 
-### Other Options
+### Tokenizer Options
```

Comment:
The parser now has a tokenizer arg section, so I modified the README to match that hierarchy.